qutip/bloch.py

```diff
@@ -112,7 +112,9 @@ def __init__(self, fig=None, axes=None, view=None, figsize=None,
                  background=False):
         # Figure and axes
         self.fig = fig
+        self._ext_fig = fig is not None
         self.axes = axes
+        self._ext_axes = axes is not None
```

Comment:
I don't see `self._ext_axes` being used anywhere.

Reply:
Thanks for catching that -- it was left over from earlier versions of the fix. Removed.